core/service/celery/celery.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from celery import Celery
+from kombu import Exchange, Queue
+
+
+def task_queue_config(cm_name: str) -> dict[str, Any]:
+    high_prefixed = f"{cm_name}:high"
+    default_prefixed = f"{cm_name}:default"
+
+    return {
+        "task_queues": (
+            Queue(high_prefixed, Exchange(high_prefixed), routing_key="high"),
+            Queue(default_prefixed, Exchange(default_prefixed), routing_key="default"),
```

Comment:
For this prototype I've defined two queues, one high priority queue, and one low priority queue. The idea is to use the high priority queue for fast running tasks, and the default queue for longer running tasks, where starvation is less of a concern. As we roll this out we may need to adjust the queues we use to be finer grained then this, but this seemed reasonable for a prototype.

Reply:
Moved into a comment in the code